openstack/keystone/values.yaml

```diff
@@ -186,7 +186,7 @@ cache:
 
 tempest:
   enabled: false
-  image: "monsoon/rally-openstack"
+  image: "ccloud/rally-openstack"
```

Comment:
rally-openstack has not yet been migrated to keppel

Reply:
i.e.: please wait  - I will see if I can find the Dockerfile/pipeline and build it respectively